backend/routes/userRoutes.js

```diff
@@ -105,19 +105,14 @@ router.patch('/location', async(req, res) => {
   res.json(updatedUser)
 })
 
-router.get('/recommendedEvents/:userId', async (req, res) => {
-  const { userId } = req.params
+const recommendTenEvents = async (user) => {
```

Comment:
Break each score into a separate function. This function is still too long.

Reply:
Will do in separate PR

Reply:
Done